.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+### Description
+
+Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.
+
+Fixes # [ISSUE]
+
+### Type of Change:
+**Delete irrelevant options.**
+
+- Code
+- Quality Assurance
+- User Interface
+- Outreach
+- Documentation
+
+**Code/Quality Assurance Only**
+- Bug fix (non-breaking change which fixes an issue)
+- This change requires a documentation update (software upgrade on readme file)
+- New feature (non-breaking change which adds functionality pre-approved by mentors)
+
+
+
+### How Has This Been Tested?
+Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.
+
+
+### Checklist:
+**Delete irrelevant options.**
+
+- [ ] My PR follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code or materials
+- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] Any dependent changes have been merged
+- [ ] I have written Kotlin Docs whenever is applicable
```

Comment:
we don't use kotlin here

Reply:
Sorry, I didn't notice. I didn't read whole template. This kind of mistake won't happen again.